src/main/kotlin/tanvd/grazi/ide/ui/rules/GraziRulesTree.kt

```diff
@@ -13,17 +13,20 @@ import com.intellij.util.ui.JBUI
 import com.intellij.util.ui.UIUtil
 import com.intellij.util.ui.tree.TreeUtil
 import org.picocontainer.Disposable
+import tanvd.grazi.GraziConfig
 import tanvd.grazi.ide.ui.panel
 import tanvd.grazi.language.Lang
 import tanvd.grazi.language.LangTool
 import java.awt.BorderLayout
+import java.awt.Component
 import java.util.*
 import javax.swing.JTree
 import javax.swing.tree.DefaultMutableTreeNode
 import javax.swing.tree.DefaultTreeModel
 
 class GraziRulesTree(selectionListener: (meta: Any) -> Unit) : Disposable {
     private val state = HashMap<String, RuleWithLang>()
+    val langs = HashSet<Lang>(GraziConfig.get().enabledLanguages)
```

Comment:
Make it private and add some method for `addLang` and `removeLang`. Otherwise GraziRulesTree exposes the whole MutableSet outside :(

Reply:
Done